src/laravel/pagseguro/Checkout/AbstractCheckout.php

```diff
@@ -78,7 +78,7 @@ public function __construct($data = [])
         $data = null;
         $this->hydrateMagic(
             [
-                'items', 'sender', 'shipping', 'currency',
+                'paymentMode', 'paymentMethod', 'creditCard', 'items', 'sender', 'shipping', 'currency',
```

Comment:
A abstração comporta o modo "clássico" de checkout, portanto a parte transparente tem que reimplementar o contrutor se for necessário.  
Isso porque na versão anterior, o contrutor poderia ter os args com os items em primeiro lugar, agora somente no quarto, e isso não se aplica ao modo clássico.